docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
       - KUBERPULT_DEX_ENABLED=false
       - KUBERPULT_GIT_NETWORK_TIMEOUT=3s
       - KUBERPULT_GIT_WRITE_COMMIT_DATA=true
+      - ALLOW_LONG_APP_NAMES=false
```

Comment:
```suggestion
      - KUBERPULT_ALLOW_LONG_APP_NAMES=false
```

Reply:
changed but set to true, or should it be only true for the `docker-compose-earthly.yml`? @sven-urbanski-freiheit-com